packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/ManageContest/steps/DetailsFormStep/utils.ts

```diff
@@ -0,0 +1,46 @@
+import * as colors from '../../../../../../../../styles/mixins/colors.scss';
+
+export const INITIAL_PERCENTAGE_VALUE = 10;
+export const MAX_WINNERS = 10;
+export const MIN_WINNERS = 1;
+
+export const prizePercentageOptions = [
+  {
+    label: '10%',
+    value: INITIAL_PERCENTAGE_VALUE,
+  },
+  {
+    label: '20%',
+    value: 20,
+  },
+  {
+    label: '30%',
+    value: 30,
+  },
+  {
+    label: '40%',
+    value: 40,
+  },
+  {
+    label: '50%',
+    value: 50,
+  },
+];
+
+export const initialPayoutStructure = [60, 25, 15];
+
+export const getPrizeColor = (index: number) => {
```

Comment:
is the `index` here for contest position holders? like `index = 0 === 1st`?

Reply:
it's for the `winners & payout` section 👇 

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/394e39b8-598d-4503-b049-70da83faf81b)